src/serde/ser_br.rs

```diff
@@ -77,10 +78,45 @@ pub fn node_to_stream_backrefs<W: io::Write>(
     Ok(())
 }
 
+pub fn node_to_bytes_backrefs_limit(
```

Comment:
I think this was a misunderstanding. it's the serialization *without* back-references that's at risk of blowing up the output buffer size, not the one where we allow back-refs.

Reply:
Arvid is correct.

Reply:
Thanks for comment. Also I discussed this with Arvid on keybase. I recognized my misunderstanding.
`LazyNode` with Allocator/NodePtr is some kind of "compressed" form because the same atom is stored only once in Allocator. This is almost the same concept with backref. When serializing this kind of `LazyNode` without backref, it does something similar to "uncompressing" the data.